tests/testthat/test-rma.R

```diff
@@ -0,0 +1,207 @@
+context("rma")
```

Comment:
Would you mind structuring the tests based on method? This is what the other tests files do. For an example you could look at: https://github.com/tidymodels/broom/blob/master/tests/testthat/test-aer.R#L33